### DISCUSSION TITLE
apakah ada shell yang kedetek tapi ga dimasukin?

### DISCUSSION THREAD

**Discussion by Stackopoy** [Arya-f4/worldshellfinder#6] [Q&A] [answered]
kira-kira gimana bang?

**Comment by Aryma-f4** [comment, marked as answer]
Iya bang ada beberapa shell yang mungkin ada yang ga saya masukin ke list kedetek, aga susah ngedatanya bang list shell apa aja yang udah kedetek, sama yang belum, biasanya kalau belum pun langsung aku kasih detectornya.
